packages/css/src/radio-button/radio-button.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../node_modules/@utrecht/components/radio-button/css";
+
+// Similar concept as Utrecht to set private variables and update them when specific state is set.
+.amsterdam-radio-button {
+  --_amsterdam-radio-button-background-color: var(--amsterdam-radio-button-background-color, currentColor);
+}
+
+.amsterdam-radio-button:checked {
+  background-image: radial-gradient(
+    circle,
+    var(--_amsterdam-radio-button-background-color, transparent) (55% - 5%),
+    currentColor 60%
+  );
+}
+
+.amsterdam-radio-button:hover {
+  box-shadow: var(--amsterdam-radio-button-hover-box-shadow);
```

Comment:
The hover `box-shadow` should be inset, I think. The shadow is currently visible for disabled buttons as well, that shouldn't be the case I think?